Cargo.toml

```diff
@@ -22,17 +22,19 @@ rustdoc-args = ["--cfg", "__time_02_docs"]
 default = ["std"]
 macros = ["time-macros"]
 local-offset = ["std", "libc", "winapi", "stdweb"]
+quickcheck-gen = ["quickcheck", "rand"]
 std = ["standback/std"]
 
 [dependencies]
 const_fn = "0.4.2"
+quickcheck = { version = "0.9", optional = true }
 rand = { version = "0.7", optional = true, default-features = false }
 serde = { version = "1", optional = true, default-features = false, features = ["derive"] }
 standback = { version = "0.2.5", default-features = false }
 time-macros = { version = "0.1", path = "time-macros", optional = true }
 
 [workspace]
-members = ["time-benchmarks", "time-macros", "time-macros-impl"]
+members = ["time-benchmarks", "time-macros", "time-macros-impl", "time-quickcheck-tests"]
```

Comment:
My main reason was to ensure that the `quickcheck-gen` feature is enabled, so it's impossible to accidentally not run the tests by not having the feature enabled. But I guess that doesn't really matter much since you run the tests against all the features in CI, so I can merge this into the base library if you prefer.

Reply:
I've already got some other things gated in the tests, so gating this would be in line with that. Obviously it wouldn't be run unless the flag is enabled, but that's to be expected.

Reply:
Done!